presto-docs/src/main/sphinx/release/release-0.234.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.234
+=============
+
+General Changes
+_______________
+* Fix query failure for cross-joining bucketed tables with ``COALESCE`` when the bucket counts are compatible but mismatched.
+* Fix query failure for window functions with ``IGNORE NULLS``.
+* Fix a regression in the new scheduler temporarily by setting the default for the configuration property ``use-legacy-scheduler`` to ``true``.
+* Improve system reliability by preventing additional queries to execute when the total number of tasks exceeds the limit, which can be
+  set by configuration property ``experimental.max-total-running-task-count-to-halt-scheduling``.
+* Improve storage efficiency for Bing tiles by storing Bing tiles as ``BIGINT``. This also reduces bucket skew in certain cases.
+* Improve the scale writer heuristics by considering overall producer buffer utilization. This can be enabled by using the session property
+  ``optimized_scale_writer_producer_buffer`` and the configuration property ``optimized-scale-writer-producer-buffer``.
+* Improve connection pooling to avoid running out of sockets.
+* Add ``KHyperLogLog`` type and related functions.
+* Add support for forcing streaming exchange for Mark Distinct using ``query.use-streaming-exchange-for-mark-distinct``.
+
+Geospatial Changes
+__________________
+* Add support for spatial joins for ``ST_Distance(p1, p2) < r``.
```

Comment:
@jagill Is `ST_Distance(p1, p2) < r` the join condition?

Reply:
It's a spatial join condition.  It is initially a cross join, but the optimizer notices certain conditions (`ST_Distance(p1, p2) < r` is one) and converts it to a spatial join.

Previously, it would only do it if `p1` and `p2` were Euclidean geometries; this change enables it for Spherical geographies as well.